### DISCUSSION TITLE
Error: Docker container error

### DISCUSSION THREAD

**Discussion by jhmjam** [ludeknovy/jtl-reporter#274] [Q&A]
![Screenshot 2024-04-15 at 11 16 15](https://github.com/ludeknovy/jtl-reporter/assets/89923453/bd74574a-14cb-4007-942d-d4029b6b7e77)

Hi I used docker and ran the compose command but one of the containers failed.

Tried removing and started again but same result.

Am i doing something wrong?

**Comment by ludeknovy** [comment]
Hi @jhmjam can you share the docker compose logs? or just db logs? Without the logs, it's impossible to tell what could be the problem.

**Reply by yuricaprini** [reply]
Hi, i have a similar problem: jtl-reporter-db keeps remaining in a waiting state.

Here the logs: 

docker logs jtl-reporter-db
********************************************************************************
WARNING: POSTGRES_HOST_AUTH_METHOD has been set to "trust". This will allow
         anyone with access to the Postgres port to access your database without
         a password, even if POSTGRES_PASSWORD is set. See PostgreSQL
         documentation about "trust":
         https://www.postgresql.org/docs/current/auth-trust.html
         In Docker's default configuration, this is effectively any other
         container on the same system.

         It is not recommended to use POSTGRES_HOST_AUTH_METHOD=trust. Replace
         it with "-e POSTGRES_PASSWORD=password" instead to set a password in
         "docker run".
********************************************************************************
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "C.UTF-8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

initdb: error: could not change permissions of directory "/home/postgres/pgdata/data": Operation not permitted
fixing permissions on existing directory /home/postgres/pgdata/data ...

There is a problem with bind mount and permission, but i do not how to solve it...

**Reply by ludeknovy** [reply]
Hi @yuricaprini !
Did you try any AI assistant to help you to debug this? In general this is not a jtl-reporter issue. This is rather about docker permission issue.
